src/core/ui/forms/MarkdownInputControls/InsertEmbedCodeButton/Iframe.ts

```diff
@@ -0,0 +1,84 @@
+import { Node } from '@tiptap/core';
+
+// We need to declare it explicitly since the iframe option is still in experimental phase.
+declare module '@tiptap/core' {
+  interface Commands<ReturnType> {
+    iframe: {
+      setIframe: (options: { src: string }) => ReturnType;
+    };
+  }
+}
+
+const defaultIframeAttributes = {
+  src: { default: null },
+  position: { default: 'absolute' },
+  top: { default: 0 },
+  left: { default: 0 },
+  width: { default: '100%' },
+  height: { default: '100%' },
+  frameborder: { default: 0 },
+  sandbox: { default: 'allow-scripts allow-same-origin allow-popups allow-presentation' },
+};
+
+export type IframeOptions = {
+  allowFullscreen: boolean;
+  HTMLAttributes: {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    [key: string]: any; // It's any by default, keep it.
```

Comment:
can we change this to `[key: string]: string;`  ? just to avoid that `any`. If it builds and we don't really need to pass anything special to the iframe...